src/composables/pools/usePoolMigration.ts

```diff
@@ -133,17 +133,22 @@ export function usePoolMigration(
   });
 
   const migrationFn = computed(() => {
-    if (migrationType.value?.type === PoolMigrationType.AAVE_BOOSTED_POOL) {
-      return migrateBoostedPool;
-    } else if (migrationType.value?.type === PoolMigrationType.STABAL3_POOL) {
-      return migrateStabal3;
-    } else if (
-      migrationType.value?.type === PoolMigrationType.STMATIC_POOL ||
-      migrationType.value?.type === PoolMigrationType.XMATIC_POOL
-    ) {
-      return migrateStables;
-    } else {
-      return migrateBoostedPool;
+    switch (migrationType.value?.type) {
+      case PoolMigrationType.AAVE_BOOSTED_POOL:
+        return migrateBoostedPool;
+
+      case PoolMigrationType.STABAL3_POOL:
+        return migrateStabal3;
+
+      case PoolMigrationType.STMATIC_POOL:
+      case PoolMigrationType.XMATIC_POOL:
+        return migrateStables;
+
+      case PoolMigrationType.MAI_POOL:
+        return migrateMaiUsd;
+
+      default:
+        return migrateBoostedPool;
```

Comment:
Again, not sure why pool migrations stuff is in this branch.

Reply:
answered in previous comment